popx/migrator.go

```diff
@@ -596,13 +586,6 @@ func (m *Migrator) exec(ctx context.Context, fn func() error) error {
 		}
 	}
 
-	if m.Connection.Dialect.Name() == "cockroach" {
-		outer := fn
-		fn = func() error {
-			return crdb.Execute(outer)
```

Comment:
From the documentation of `crdb.Execute`:

> It is used to add retry handling to the execution of a single statement. If a multi-statement transaction is being run, use ExecuteTx instead.

But as the transaction is started within, we don't need to retry anything at this point.

Reply:
With my recent auto-commit changes, we should add retry to migrations that run without transactions (aka go migrations and auto commit transactions).